### ISSUE TITLE
Migration failing when trying to install swarfarm on Ubuntu 18.04

### ISSUE THREAD

**Issue by aerendil** [swarfarm/swarfarm#137]
Hey,

I have been trying to deploy a dev installation of swarfarm on Ubuntu 18.04. I am currently seeing an error when trying to run the migration:

```
Traceback (most recent call last):
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedTable: relation "bestiary_scalingstat" does not exist
LINE 1: ..._desc", "bestiary_scalingstat"."description" FROM "bestiary_...
```

Full traceback is here:
```
Traceback (most recent call last):
  File "manage.py", line 12, in <module>
    execute_from_command_line(sys.argv)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/core/management/base.py", line 361, in execute
    self.check()
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/core/management/base.py", line 390, in check
    include_deployment_checks=include_deployment_checks,
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/core/management/commands/migrate.py", line 65, in _run_checks
    issues.extend(super()._run_checks(**kwargs))
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/core/management/base.py", line 377, in _run_checks
    return checks.run_checks(**kwargs)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/core/checks/registry.py", line 72, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/core/checks/urls.py", line 13, in check_url_config
    return check_resolver(resolver)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/core/checks/urls.py", line 23, in check_resolver
    return check_method()
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/urls/resolvers.py", line 399, in check
    for pattern in self.url_patterns:
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/utils/functional.py", line 80, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/urls/resolvers.py", line 584, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/utils/functional.py", line 80, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/urls/resolvers.py", line 577, in urlconf_module
    return import_module(self.urlconf_name)
  File "/root/.pyenv/versions/3.6.8/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/root/swarfarm/swarfarm/urls.py", line 28, in <module>
    url(r'^api/v2/', include('apiv2.urls', namespace='v2')),
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/root/.pyenv/versions/3.6.8/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/root/swarfarm/apiv2/urls.py", line 8, in <module>
    from data_log.api_urls import router as data_log_router
  File "/root/swarfarm/data_log/api_urls.py", line 3, in <module>
    from . import views
  File "/root/swarfarm/data_log/views.py", line 8, in <module>
    from .game_commands import active_log_commands, accepted_api_params
  File "/root/swarfarm/data_log/game_commands.py", line 5, in <module>
    from bestiary.parse.dungeons import dispatch_dungeon_wave_parse
  File "/root/swarfarm/bestiary/parse/__init__.py", line 5, in <module>
    from .skills import skills, homonculus_skills
  File "/root/swarfarm/bestiary/parse/skills.py", line 169, in <module>
    _defense = ScalingStat.objects.get(com2us_desc='DEF')
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/db/models/query.py", line 402, in get
    num = len(clone)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/db/models/query.py", line 256, in __len__
    self._fetch_all()
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/db/models/query.py", line 1242, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/db/models/query.py", line 55, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1140, in execute_sql
    cursor.execute(sql, params)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/db/backends/utils.py", line 99, in execute
    return super().execute(sql, params)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/db/backends/utils.py", line 76, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/root/.pyenv/versions/swarfarm-3.6.8/lib/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: relation "bestiary_scalingstat" does not exist
LINE 1: ..._desc", "bestiary_scalingstat"."description" FROM "bestiary_...
```
Let me know if you have any idea how to solve this

**Comment by aerendil** [comment]
Looks like I managed to get the migrations to run by commenting out 3 lines in 
bestiary/parse/skills.py
```
169 _defense = ScalingStat.objects.get(com2us_desc='DEF')
170 _target_hp = ScalingStat.objects.get(com2us_desc='TARGET_TOT_HP')
171 _max_hp = ScalingStat.objects.get(com2us_desc='ATTACK_TOT_HP')
```

**Comment by PeteAndersen** [comment]
Thanks, I didn't realize those would cause an issue when running management commands.

**Comment by PeteAndersen** [comment]
Fixed with 8e63743158daca4faab74696cf5e41dff2850986
